includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -2921,20 +2921,4 @@ public function filter_my_account_my_orders_actions( $actions, $order ) {
 		}
 		return $actions;
 	}
-
-	/**
-	 * Filter the order received text for Amazon Pay orders, including the delayed confirmation information.
-	 *
-	 * @param string $text Default text.
-	 * @param WC_Order $order Order data.
-	 * @return string
-	 */
-	public function filter_thankyou_order_received_text( $text, $order ) {
-		if ( $order->get_payment_method_title() === 'Amazon Pay (Stripe)' && $order->has_status( 'pending' ) ) {
-			$text .= '<p class="woocommerce-info">';
-			$text .= esc_html( 'The payment is being processed and it might take a few minutes before it\'s confirmed.' );
-			$text .= '</p>';
-		}
-		return $text;
-	}
```

Comment:
I am removing this as payments are no longer "delayed". The delay was caused by missing code for processing the response, and has been fixed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3913